### ISSUE TITLE
ncats-webui service starts up successfully, but systemctl reports failure

### ISSUE THREAD

**Issue by dav3r** [cisagov/cyhy_amis#206]
# 🐛 Bug Report

When the `ncats-webui` service starts up on the `dashboard` instance, the `ncats-webui_web-server_1` docker container starts up correctly and the dashboard functions as expected, however `systemctl` reports that the `ncats-webui` service is in a `failed` state.

## To Reproduce

- `ssh` to a `dashboard` instance that started up successfully
- `systemctl status ncats-webui.service`

## Expected behavior

We expect to see `Active: active (running)`, rather than `Active: failed (Result: start-limit-hit)`.

## `systemctl status` output

```bash
dashboard:~$ systemctl status ncats-webui.service 
● ncats-webui.service - ncats-webui service
   Loaded: loaded (/etc/systemd/system/ncats-webui.service; static; vendor preset: enabled)
   Active: failed (Result: start-limit-hit) since Thu 2019-05-30 21:44:01 UTC; 28min ago
  Process: 2492 ExecStart=/usr/bin/make -C /var/cyhy/ncats-webui/ dev-start (code=exited, status=0/SUCCESS)
 Main PID: 2492 (code=exited, status=0/SUCCESS)
      CPU: 650ms
```

**Comment by dav3r** [comment]
@KyleEvers - is there any update on this?

**Comment by KyleEvers** [comment]
Deploying a test fix in my workspace now

**Comment by KyleEvers** [comment]
The error was being caused by systemd not having insight into the docker container once it started. Because the service was set to "restart=always" with a "RestartSec=1" it would constantly restart the service until the "start-limit-hit" occurred and the service failed. The restart value was changed to only "on-failure" and the "RestartSec" was change to 10 to avoid this endless restart loop but still retain the logging of the service startup. This bug has been fixed in PR #212
